go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/saintfish/chardet v0.0.0-20120816061221-3af4cd4741ca // indirect
 	github.com/sergi/go-diff v1.1.0 // indirect
 	github.com/shiena/ansicolor v0.0.0-20200904210342-c7312218db18 // indirect
+	github.com/siddontang/go v0.0.0-20170517070808-cb568a3e5cc0
```

Comment:
Is this package neccessary? I don't see you use it anywhere.

Reply:
@gutakk not sure why it's there, remove in [534747f](https://github.com/carryall/go-google-scraper-challenge/pull/42/commits/534747f76a4fd6521e861eaf56339f43d86408c9) 😄